librz/core/cmd/cmd_print.c

```diff
@@ -5409,9 +5408,10 @@ static void analysis_stats_json_info(RzCore *core, RzCoreAnalysisStats *as, RzCo
 }
 
 static void analysis_stats_table_info(RzCore *core, RzCoreAnalysisStats *as, RzCoreAnalysisStatsItem *sitem, ut64 blockidx, RzCmdStateOutput *state) {
+	char tmpbuf[20] = { 0 };
 	ut64 at = rz_core_analysis_stats_get_block_from(as, blockidx);
 	if ((sitem->flags) || (sitem->functions) || (sitem->comments) || (sitem->symbols) || (sitem->strings)) {
-		rz_table_add_rowf(state->d.t, "sddddd", sdb_fmt("0x%09" PFMT64x "", at), sitem->flags,
+		rz_table_add_rowf(state->d.t, "sddddd", rz_strf(tmpbuf, "0x%09" PFMT64x "", at), sitem->flags,
```

Comment:
i think the `rz_table_add_rowf` accepts `x` which would be better for an address/offset